libs/perun/components/src/lib/services-status-list/services-status-list.component.html

```diff
@@ -0,0 +1,80 @@
+                        [checked]="selection.hasValue() && isAllSelected()"
+                        [indeterminate]="selection.hasValue() && !isAllSelected()"
+                        [aria-label]="checkboxLabel()">
+          </mat-checkbox>
+        </th>
+        <td mat-cell *matCellDef="let serviceState" class="static-column-size">
+          <mat-checkbox color="primary"
+                        (click)="$event.stopPropagation()"
+                        (change)="$event ? selection.toggle(serviceState) : null"
+                        [checked]="selection.isSelected(serviceState)"
+                        [aria-label]="checkboxLabel(serviceState)">
+          </mat-checkbox>
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="id">
+        <th mat-header-cell *matHeaderCellDef
+            mat-sort-header>{{'SHARED_LIB.PERUN.COMPONENTS.SERVICES_STATUS_LIST.TASK_ID' | translate}}</th>
+        <td mat-cell class="static-column-size" *matCellDef="let serviceState">{{serviceState.task.id}}</td>
+      </ng-container>
+      <ng-container matColumnDef="serviceName">
+        <th mat-header-cell *matHeaderCellDef
+            mat-sort-header>{{'SHARED_LIB.PERUN.COMPONENTS.SERVICES_STATUS_LIST.SERVICE_NAME' | translate}}</th>
+        <td mat-cell *matCellDef="let serviceState">{{serviceState.service.name}}</td>
+      </ng-container>
+      <ng-container matColumnDef="status">
+        <th mat-header-cell *matHeaderCellDef>{{'SHARED_LIB.PERUN.COMPONENTS.SERVICES_STATUS_LIST.STATUS' | translate}}</th>
+        <td mat-cell *matCellDef="let serviceState">{{serviceState.task.status}}</td>
+      </ng-container>
+      <ng-container matColumnDef="blocked">
+        <th mat-header-cell *matHeaderCellDef>{{'SHARED_LIB.PERUN.COMPONENTS.SERVICES_STATUS_LIST.BLOCKED' | translate}}</th>
+        <td mat-cell *matCellDef="let serviceState">{{serviceState | serviceStateBlockedToString}}</td>
+      </ng-container>
+      <ng-container matColumnDef="started">
+        <th mat-header-cell *matHeaderCellDef
+            mat-sort-header>{{'SHARED_LIB.PERUN.COMPONENTS.SERVICES_STATUS_LIST.STARTED' | translate}}</th>
+        <td mat-cell *matCellDef="let serviceState">{{serviceState.task.startTime | millisecondsFromEpochToDate}}</td>
+      </ng-container>
+      <ng-container matColumnDef="ended">
+        <th mat-header-cell *matHeaderCellDef
+            mat-sort-header>{{'SHARED_LIB.PERUN.COMPONENTS.SERVICES_STATUS_LIST.ENDED' | translate}}</th>
+        <td mat-cell *matCellDef="let serviceState">{{serviceState.task.endTime | millisecondsFromEpochToDate}}</td>
+      </ng-container>
+
+      <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+      <tr
+          mat-row
+          *matRowDef="let serviceState; columns: displayedColumns;"
+          [class.disable-outline]="disableRouting"
+          [routerLink]="disableRouting ? null : ['/facilities', serviceState.facility.id, 'services-status']">
```

Comment:
can you please set disableRouting for true, while dont have implemented the page for task results?